Modules/_asynciomodule.c

```diff
@@ -1757,17 +1753,19 @@ TaskObj_finalize(TaskObj *task)
 
     func = _PyObject_GetAttrId(task->task_loop, &PyId_call_exception_handler);
     if (func != NULL) {
-        res = PyObject_CallFunctionObjArgs(func, context, NULL);
+        PyObject *res = PyObject_CallFunctionObjArgs(func, context, NULL);
```

Comment:
This would change the semantic. Currently if the "call_exception_handler" method is not defined the code does nothing. `PyObject_CallMethodObjArgs()` would raise an error.